tests/src/python/test_qgstiledmeshboundingvolume.py

```diff
@@ -113,8 +113,8 @@ def test_box(self):
         self.assertAlmostEqual(bounds.zMaximum(), 3153.6759909, 3)
 
         geometry_2d = volume.as2DGeometry(transform)
-        self.assertEqual(geometry_2d.asWkt(3),
-                         'Polygon ((149.558 -33.421, 149.569 -33.412, 149.586 -33.428, 149.597 -33.42, 149.558 -33.421))')
+        self.assertEqual(geometry_2d.asWkt(5),
+                         'Polygon ((149.58608 -33.44312, 149.55826 -33.43557, 149.55826 -33.40547, 149.56915 -33.39692, 149.59696 -33.40445, 149.59696 -33.43455, 149.58608 -33.44312))')
```

Comment:
Why these differences? Nevertheless, seems ok.

Reply:
Why increasing from 5 decimals? I did that because 3 wasn't enough to meaningfully visualise the result -- the rounding was too extreme.

Reply:
Yes, but "149.558 -33.421" to "149.58608 -33.44312" it's not only a rounding problem, the result is slighty different. But this is understandable, given the use of convex hulls (and therefore a better result).